demo/components/http-endpoint/http-endpoint-service.js

```diff
@@ -1,6 +1,6 @@
 app.service('HttpEndpoint', function($q, $http, $timeout, GENERAL_CONFIG) {  
   var endpointBaseUrl = GENERAL_CONFIG.END_POINT_BASE_URL;
-  var endpoint = endpointBaseUrl + 'namespace/test/sparql';
+  var endpoint = endpointBaseUrl + 'namespace/kb/sparql';
```

Comment:
Just wondering: is `kb` the default namespace?

Reply:
Yes, kb is the default one